docs/pages/guides/spend-permissions/overview.mdx

```diff
@@ -0,0 +1,70 @@
+# Spend Permissions
+
+Spend Permissions enable third-party signers to spend assets (native and ERC-20 tokens) from user wallets. 
+This unlocks the following use cases: // TODO make note about reduced signing
+- frequent in-game spending
+- subscription payments
+- auto-renewals
+- automated or event-based trading
+- micropayment streaming
+
+Differences between Spend Permissions and ERC-20 permits:
+- Spend Permissions **supports all ERC-20 assets**, whereas permits only support ERC-20s that implement the permit standard  
+- Spend Permissions **enable spending native tokens**  
+- Spend Permissions offers granular **controls for recurrence**  
+- The logic that controls asset movements is **implemented in the user's wallet**, not the asset contract  
+
+
+## The `SpendPermission` details
+A spend permission is defined by the following parameters
+
+```solidity
+struct SpendPermission {
+    address account;
+    address spender;
+    address token;
+    uint160 allowance;
+    uint48 period;
+    uint48 start;
+    uint48 end;
+    uint256 salt;
+    bytes extraData;
+}
+```
+
+
+## Approving
+
+A user approves a spend permission by signing an [ERC-712](https://eips.ethereum.org/EIPS/eip-712) typed object that contains
+the spend permission properties. This signature and the corresponding spend permission details are then submitted to 
+`SpendPermissionManager.approveWithSignature` to approve the spend permission onchain.
+
+Multiple spend permissions can be batched and approved with a single signature by constructing a batch spend permission object
+and approving with `SpendPermissionManager.approveBatchWithSignature`.
+
+
+## Revoking
+
+Users can revoke permissions at any time by calling `SpendPermissionManager.revoke`. Calls to revoke can be batched via 
+`CoinbaseSmartWallet.executeBatch`. Once a spend permission has been revoked, it can never be re-approved. 
+
+If the user wants to re-authorize the spend permission, the spender will need to generate a new spend permission 
+that has a unique hash from the original spend permission. If the details of the new spend permission are identical
+to the revoked permission, the `salt` field of the permission should be used to generate a unique hash.
```

Comment:
imo this is a little too technical for the overview section, would make sense as a notice on the getting started page though